conf/websocket.conf

```diff
@@ -59,7 +59,7 @@ authenticationProviders=
 # Enforce authorization
 authorizationEnabled=false
 
-# Actions that can be authorized by using permitted role name which contains wildcard
+# Allow wildcard matching in authorization
```

Comment:
does it make sense if we can add small description about wildcard position, eg:
`Allow wildcard matching in authorization (wildcard matching only applicable if wildcard-char : * presents at first or last position eg: *.pulsar.service, pulsar.service.*)`  ???